src/docs/framework-integration/angular.md

```diff
@@ -39,211 +43,179 @@ top-most-directory/
+Install the `@stencil/angular-output-target` dependency to your Stencil component library package.
 
-`npm i @stencil/angular-output-target`
-
-or
-
-`yarn add @stencil/angular-output-target`
+```bash
+npm install @stencil/angular-output-target --save-dev
+```
 
-### Add the Angular Wrapper Function to your Stencil Component Library
-With the Angular Output Target package installed, we can now configure our Stencil Component Library to build our Angular wrapped components. In the `stencil.config.ts` file of your Stencil component library, add the Angular wrapper function. 
+In your project's `stencil.config.ts`, add the `angularOutputTarget` configuration to the `outputTargets` array:
 
-Within your Stencil’s config file, you can import the Angular wrapper function for use within the outputTargets array. If you copy and paste this, ensure you update the “my-workspace” and “my-lib” to coincide with what you named your component library, as well as to update the `componentCorePackage` to the name of your Stencil component library.
+```ts
+import { angularOutputTarget } from '@stencil/angular-output-target';
 
-```tsx
-import { angularOutputTarget as angular } from '@stencil/angular-output-target';
 export const config: Config = {
-  namespace: 'demo',
+  namespace: '',
   outputTargets: [
-    angular({
-      componentCorePackage: `your-stencil-library-name`,
-      directivesProxyFile: `../your-angular-workspace-name/projects/your-angular-library-name/src/lib/stencil-generated/components.ts`
+    angularOutputTarget({
+      componentCorePackage: 'your-stencil-library-package-name',
+      directivesProxyFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
     }),
-    {
-      type: 'dist',
-      esmLoaderPath: '../loader',
-    },
-    {
-      type: 'dist-custom-elements',
-    },
   ],
 };
 ```
 
-Once you have added this and have `directivesProxyFile` pointing to the correct directory of your Angular component library, you can build Stencil. 
+> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
 
-`npm run build`
+You can now build your Stencil component library to generate the component wrappers.
 
-Or
+```bash
+npm run build
+```
 
-`yarn build`
+If the build is successful, you will now have contents in the file specified in `directivesProxyFile`.
 
-Once the build is complete, you will see new files in your Angular component library's directory!
+You can now finally import and export the generated component wrappers for your component library. For example, in your main `AppModule` you may do the following:
```

Comment:
Should this be the main `AppModule` for our Angular application, or for our component library? I'm a little confused here